amundsen_application/api/issue/issue.py

```diff
@@ -0,0 +1,78 @@
+from flask import current_app as app
+from flask import jsonify, make_response, Response
+from flask_restful import Resource, reqparse
+from http import HTTPStatus
+import logging
+
+from amundsen_application.issue_tracker_clients import get_issue_tracker_client
+from amundsen_application.issue_tracker_clients.issue_exceptions import IssueConfigurationException
+from jira import JIRAError
+
+LOGGER = logging.getLogger(__name__)
+
+
+class IssuesAPI(Resource):
+    def __init__(self) -> None:
+        self.reqparse = reqparse.RequestParser()
+        # declared as none initially so that it doesn't throw configuration errors until a call is made
+        self.client = None
+
+    def get(self) -> Response:
+        """
+        Given a table key, returns all tickets containing that key. Returns an empty array if none exist
+        :return: List of tickets
+        """
+        try:
+            self.client = get_issue_tracker_client()
+            if not app.config['ISSUE_TRACKER_CLIENT_ENABLED']:
+                message = 'Issuing tracking is not enabled. Request was accepted but no issue will be returned.'
+                logging.exception(message)
+                return make_response(jsonify({'msg': message}), HTTPStatus.ACCEPTED)
+
+            self.reqparse.add_argument('key', 'Request requires a key', location='args')
+            args = self.reqparse.parse_args()
+            response = self.client.get_issues(args['key'])
+            return make_response(jsonify({'issues': response}), HTTPStatus.OK)
+
+        except IssueConfigurationException as e:
+            message = 'Encountered exception: ' + str(e)
+            logging.exception(message)
+            return make_response(jsonify({'msg': message}), HTTPStatus.NOT_IMPLEMENTED)
+        except (JIRAError, Exception) as e:
```

Comment:
I don't think we need to expose `JIRAError` here, do we?

Reply:
whoops yeah good point!